packages/es-components/src/components/controls/answer-group/AnswerGroup.md

```diff
@@ -0,0 +1,58 @@
+Group `RadioButton` components together using a `RadioGroup`. Any additional props passed to `RadioGroup` will propagate to each individual `RadioButton`.
```

Comment:
Don't forget to replace references to `RadioGroup`. :)

Reply:
Replaced with `AnswerGroup` and `AnswerButton`